packages/block-editor/src/components/block-title/stories/index.story.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * WordPress dependencies
+ */
+import { registerCoreBlocks } from '@wordpress/block-library';
+import { createBlock } from '@wordpress/blocks';
+import { BlockEditorProvider } from '@wordpress/block-editor';
```

Comment:
Since this component is an internal dependency, we write it as follows:

```JavaScript
/**
 * Internal dependencies
 */
import { ExperimentalBlockEditorProvider } from '../../provider';
```